hppc/src/main/java/com/carrotsearch/hppc/ArraySizingStrategy.java

```diff
@@ -24,4 +24,14 @@
    */
   int grow(int currentBufferLength, int elementsCount, int expectedAdditions)
       throws BufferAllocationException;
+
+  @Override
+  default long ramBytesAllocated() {
```

Comment:
I would not use default implementation for the two methods here, to be sure implementing classes take care of them.
There could be a small risk of incompatibility with custom implementations of this interface. But if with default impl here the resulting ramBytesAllocated would be wrong if they don't realize they should override it.

Reply:
Good point!
I'll change it.